db/migrate/20160927071039_create_notification_types.rb

```diff
@@ -0,0 +1,13 @@
+class CreateNotificationTypes < ActiveRecord::Migration
+  def change
+    create_table :notification_types do |t|
+      t.string :name, index: true, unique: true
```

Comment:
From what I see on the UI side http://www.patternfly.org/pattern-library/communication/notification-drawer/#/api only the `group` and `message` are displayed, therefore the name would only be used internally, and the table already has an id